lib/active_model/serializer/reflection.rb

```diff
@@ -103,7 +103,11 @@ def build_association(subject, parent_serializer_options)
           begin
             serializer = serializer_class.new(
               association_value,
-              serializer_options(subject, parent_serializer_options, reflection_options)
+              serializer_options(
+                subject,
+                parent_serializer_options,
+                reflection_options.merge(options)
```

Comment:
I'm guessing we should rename this variable to `field_options` so it's clear where it's coming from. It took me a moment.